codebuild/cd/test-fips-branch.sh

```diff
@@ -9,12 +9,12 @@ cd ./crt/aws-lc || exit 1
 # Get the current commit hash
 current_commit=$(git rev-parse HEAD)
 
-# Check if the current commit is from the "fips-2022-11-02" branch
-if git merge-base --is-ancestor "$current_commit" "origin/fips-2022-11-02"; then
-    echo "Current aws-lc commit is from the 'fips-2022-11-02' branch"
+# Check if the current commit is from the "AWS-LC-FIPS-3.0.0" branch
+if git merge-base --is-ancestor "$current_commit" "AWS-LC-FIPS-3.0.0"; then
```

Comment:
"AWS-LC-FIPS-3.0.0" is a tag, not a branch
the branch is "fips-2024-09-27" right?

Reply:
Yes

Reply:
Suggesting fix in next comment...